src/player.cpp

```diff
@@ -729,53 +780,19 @@ BETTER_SMS_FOR_CALLBACK void initMario(TMario *player, bool isMario) {
             JKRArcFinder *finder = JKRFileLoader::findFirstFile("/mario/custom");
             if(finder != 0x0) {
```

Comment:
Change these comparisons from 0x0 to nullptr which is more correct and explicit to read.